nx-X11/programs/Xserver/fb/fbpict.c

```diff
@@ -901,11 +885,6 @@ fbComposite (CARD8      op,
 			switch (pDst->format) {
 			case PICT_r5g6b5:
 			case PICT_b5g6r5:
-#ifdef USE_MMX
-			    if (fbHaveMMX())
-				func = fbCompositeSolidMask_nx8x0565mmx;
-			    else
-#endif
 				func = fbCompositeSolidMask_nx8x0565;
```

Comment:
Theoretically, you could move the the call following the `USE_MMX` block one indentation level to the left, but that's really just an optical thing.

Would apply to all the following change blocks as well.